### ISSUE TITLE
SteamProtocol Tab Missing for Requests – Potential Host Header Issue

### ISSUE THREAD

**Issue by andyspectre** [njbooher/burp-steam-protocol#2]
Hi @njbooher,

I've been exploring the extension for a bit and noticed that it handles HTTP responses without issues. However, there seems to be a problem with requests—the SteamProtocol tab does not appear in the message editor, making it impossible to decode and modify protobufs in requests.

After checking the extension's logs, I found that the Host header in HttpRequestResponse can sometimes be null, causing errors. To confirm, I temporarily set the value to "api.steampowered.com," which resolved the issue. However, I believe there should be a more robust solution.

I'm continuing to investigate this, and if you’re open to it, I may submit a PR soon. I'd appreciate any feedback or guidance on how you would prefer this issue to be handled.

Thanks for your time and for all your hard work on this great extension!

**Comment by njbooher** [comment]
Does it work better if you use the jar from this workflow run?
https://github.com/njbooher/burp-steam-protocol/actions/runs/11898235611

**Comment by andyspectre** [comment]
Not really. Now it correctly shows the extension tab in the request editor, but the content is empty, as well as the reponses one.

![Screenshot_20241123_192139](https://github.com/user-attachments/assets/ef56cb6f-a626-4b87-9022-15287d1486bb)

**Comment by njbooher** [comment]
If you figure out the cause and want to make a PR, I'll take a look.

**Comment by andyspectre** [comment]
Ok I ll try to figure it out. Thanks.
